### DISCUSSION TITLE
原来这个绳网不只是zzz啊

### DISCUSSION THREAD

**Discussion by kiana423-hub** [share121/inter-knot#5398] [原神]
希望这个平台没有无脑的黑子

**Comment by ZhaoWuXin** [comment]
不好说哦，扯上了米的基本上都有

**Reply by kiana423-hub** [reply]
但应该没有机器人

**Comment by March-777th** [comment]
致敬传奇利刃ghost，一人删了一千条帖子<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/40f465ef5312e4ba0b19c596b9999cef_8760502457253372865.png" alt="三月七 无语" width="75" >这么个小角落都要爆破
